credentials/apps/edx_credentials_extensions/edly_credentials_app/api/serializers.py

```diff
@@ -0,0 +1,67 @@
+"""
+Serializers for edly_api
+"""
+from django.contrib.sites.models import Site
+
+from rest_framework import serializers
+
+from credentials.apps.credentials.models import ProgramCertificate, Signatory
+from credentials.apps.edx_credentials_extensions.edly_credentials_app.api.fields import Base64ImageField
+
+
+class SignatorySerializer(serializers.ModelSerializer):
+    """
+    Serializer for Signatories
+    """
+    image = Base64ImageField(max_length=None, use_url=True)
+
+    class Meta:
+        model = Signatory
+        fields = ['id', 'name', 'title', 'image']
```

Comment:
Where is this `id` used?

Reply:
This is a read-only field that will be sent back to update the endpoint which would help us in finding the exact Signatory. Kindly see the code attached below.
https://github.com/edly-io/credentials/pull/5/files#diff-c363da0b14ed89bb4139aeb89832cf57b5e5f70732ea4df8cda7fea380221a42R49